packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -109,6 +109,7 @@ The object's type is \`TableValidityState\`, and it contains the following boole
 -   \`invalidRecordId\`: \`true\` when a record was found where \`id-field-name\` did not refer to a value of type \`string\`
```

Comment:
While you're here, could you update line 105? It looks like the type isn't called `TableValidityState` anymore, just `TableValidity`. There's a similar incorrect name in `packages/nimble-components/src/table/specs/README.md`

Reply:
Sure